frontend/src/features/idir/search/components/IDIRPermitSearchRowActions.tsx

```diff
@@ -13,11 +13,9 @@ interface PermitAction {
 
 const PERMIT_ACTIONS: PermitAction[]  = [
   {
-    actionName: "Amend",
-    isAuthorized: (isExpired: boolean, userAuthGroup?: string) => 
-      !isExpired && (
-        userAuthGroup === USER_AUTH_GROUP.PPCCLERK || userAuthGroup === USER_AUTH_GROUP.SYSADMIN
-      ),
+    actionName: "Resend",
```

Comment:
consider an `enum` for the actions

Reply:
I'll add for these and other action components

Reply:
Added